test_data/sync.json

```diff
@@ -200,31 +197,7 @@
                                 "prev_sender": "@example:localhost",
                                 "replaces_state": "$152034819067QWJxM:localhost"
                             }
-                        },
-                        {
-                            "content": {
-                              "membership": "leave",
-                              "reason": "offline",
-                              "avatar_url": "avatar.com",
-                              "displayname": "example"
-                            },
-                            "event_id": "$1585345508297748AIUBh:matrix.org",
-                            "origin_server_ts": 1585345508223,
-                            "sender": "@example:localhost",
-                            "state_key": "@example:localhost",
-                            "type": "m.room.member",
-                            "unsigned": {
-                              "replaces_state": "$1585345354296486IGZfp:localhost",
-                              "prev_content": {
-                                "avatar_url": "avatar.com",
-                                "displayname": "example",
-                                "membership": "join"
-                              },
-                              "prev_sender": "@example2:localhost",
-                              "age": 6992
-                            },
-                            "room_id": "!roomid:room.com"
```

Comment:
There are no room_id's in sync responses right?

Reply:
They are not required per spec and Synapse doesn't include them, now other servers might include them

Reply:
Yeah it's not great that serde fails when it encounters a unknown key in the JSON. In the ruma-events matrix-sdk actually uses room_id is `Option<RoomId>` maybe that's the way to do it?